### PR TITLE
CCD-3694 CCD-3836 test definitions for FTAs for CCD-3675 & CCD-3798

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [CCD-3675](https://tools.hmcts.net/jira/browse/CCD-3675) _"Global Search - Access Process should take into account mapped access profiles"_
   [CCD-3694](https://tools.hmcts.net/jira/browse/CCD-3694) _"FTAs for Global Search - Access Process CCD-3675"_

   [CCD-3798](https://tools.hmcts.net/jira/browse/CCD-3798) _"ACCESS_GRANTED metadata needs to consider role assignments giving access to cases"_
   [CCD-3836](https://tools.hmcts.net/jira/browse/CCD-3836) _"FTAs for ACCESS_GRANTED metadata changes CCD-3798"_

### Change description ###

Test definitions and CCD Roles for FTAs for CCD-3675 & CCD-3798. These new CCD roles require hmcts/befta-fw#192 to allow them to be published.

> Test PRs in main repos: 
> * hmcts/ccd-user-profile-api#509
> * hmcts/ccd-definition-store-api#1309
> * hmcts/ccd-data-store-api#2144

> NB: This PR builds upon the #55.

> NB: This change is to support FTAs for:
> * hmcts/ccd-data-store-api#2105
> * hmcts/ccd-data-store-api#2129

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.6"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.7-CCD-3694_CCD-3836"
```
Comment (mattnayler) [inline build.gradle:9]: This should be updated to match final version number before being merged to `master`.

File: build.gradle [hunk 2]
```diff
@@ -137,7 +137,7 @@ publishing {
 wrapper.gradleVersion = '2.10'
 
 dependencies {
-    compile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6'
+    compile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.7-CCD-3694_CCD-3836'
```
Comment (mattnayler) [inline build.gradle:140]: Should be updated with proper release number once hmcts/befta-fw#192 is released.

File: build.gradle [hunk 3]
```diff
@@ -6,7 +6,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.7"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "7.19.10-CCD-3694_CCD-3836"
```
Comment (mattnayler) [inline build.gradle:9]: This should be updated to match final version number before being merged to `master`.

File: build.gradle (hunk 1 above)
Comment (mattnayler) [reply build.gradle]: Updated see https://github.com/hmcts/ccd-test-definitions/pull/57#discussion_r1029528180

**Review by marianadpereira — APPROVED** [review]

**Pull request merged**